src/merchant-config.d.ts

```diff
@@ -18,4 +18,8 @@ export interface MerchantConfig {
    * Used to run in test-mode. The requests will be sent to Payex' Test API.
    */
   testMode?: boolean;
+  /**
+   * Override for PayEx API. Replaces testMode to customise environment.
+   */
+  callbackOverride?: string;
```

Comment:
```suggestion
  /**
   * Set URL for all methods. Cannot be used with `testMode`.
   */
  baseUrl?: string;
```

I think we should call this `baseUrl`, as it's more widely used for this :)